src/Utility.cc

```diff
@@ -1101,7 +1104,11 @@ namespace ignition
         }
 
         out << "<?xml version='1.0'?>\n"
-            << "  <model>\n";
+            << "  <model>\n"
+            << "    <sdf version='"
```

Comment:
The `<sdf>` element should go outside the `<model>` / `<world>`

Reply:
It does for `*.sdf` files but not `*.config` files. Here are some examples:
- https://github.com/gazebosim/sdformat/blob/sdf12/test/integration/model/box/model.config#L5
- https://github.com/gazebosim/sdformat/blob/sdf12/test/integration/model/robot/model.config#L4

Reply:
Ah you're right, I got them mixed up 🙃